azext_iot/_help.py

```diff
@@ -1216,3 +1216,18 @@
     type: command
     short-summary: Delete a device registration in an Azure IoT Hub Device Provisioning Service.
 """
+
+helps[
+    "iot dps device-key"
+] = """
+    type: group
+    short-summary: Create device SAS key
```

Comment:
nit: Create->Manage (since later we might want show/delete/etc)

Reply:
You know, this command is entirely related to generating a derived key from a DPS enrollment group. The `az iot dps enrollment-group` commands already exist for show/create/delete/list -- is there anything else coming for this command group? If not I'm wondering about putting the device-key create command under enrollment groups.

Reply:
This was more of a lets have a dialog comment